Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -242,172 +254,220 @@ edgebridgetutorialapp://
+With this, the migration from an existing Analytics implementation to sending data via Edge Network to Analytics, powered by Edge Bridge, is complete!
 
-> **Note**
-> The following documentation provides a comprehensive overview of the Data Prep capabilities:
-> - [Data Prep overview](https://experienceleague.adobe.com/docs/experience-platform/data-prep/home.html)
-> - [Data Prep mapping functions](https://experienceleague.adobe.com/docs/experience-platform/data-prep/functions.html)
-> - [Handling data formats with Data Prep](https://experienceleague.adobe.com/docs/experience-platform/data-prep/data-handling.html)
->
+## Optional - Data Prep for Data Collection mapping
+Edge Bridge transforms Analytics track events into an Edge Network event format suitable for Analytics. To use this same event data in other Experience Platform applications, it must first be mapped to the [Experience Data Model (XDM)](https://experienceleague.adobe.com/docs/experience-platform/xdm/home.html) specification using [Data Prep for Data Collection](https://experienceleague.adobe.com/en/docs/experience-platform/datastreams/data-prep).
 
-</p></details>
+This section covers how to map event data sent from Edge Bridge in the Data Collection UI.
 
-To open the data prep mapper: 
-1. Select **Datastreams** (**1**) in the left-side navigation panel.
-2. Select your datastream (**2**).
+To open the Data Prep mapper:
+1. Log in to the [Adobe Experience Platfom](https://experience.adobe.com/#/platform).
+2. In the left-side navigation panel under **DATA COLLECTION**, select **Datastreams** (**1**).
+3. Select your datastream (**2**).
 
 <img src="../assets/edge-bridge-tutorial/data-prep/datastreams-mapper-nav-1.png" alt="Datastream mapper navigation" width="1100"/>  
 
-3. Select **Edit Mapping** (**1**) in the right-side navigation panel.  
+4. In the right-side navigation panel, select **Edit Mapping** (**1**).  
 
 <img src="../assets/edge-bridge-tutorial/data-prep/datastreams-mapper-nav-2.png" alt="Datastream mapper navigation 2" width="1100"/>
 
-Currently, the data mapper UI only allows for one JSON payload to be mapped per datastream. This means for a given datastream, all of the potential event payloads need to be merged so that they can be mapped at once. Note that Data Prep requires that the **data** and **xdm** objects are top-level objects in the provided JSON source.
+> [!IMPORTANT]
+> Data Prep allows only one consolidated mapping configuration per datastream, meaning all potential event payloads sent through a specific datastream must be merged for simultaneous mapping. Note that Data Prep requires `data` and `xdm` to be top-level objects in the source JSON.
+
+We will cover 3 mapping use cases:
+1. Analytics-formatted product strings
+2. Analytics-formatted events strings
+3. Custom `contextData` keys
```

Comment:
- let's reorder these mapping use-cases to be in the same order as the detailed steps described below.
- are you planning to add an example with events strings too or should we consolidate the first two into Analytics- formatted products/events strings